### PR TITLE
add clear() method to Table

### PR DESCRIPTION
**Pull request by JerryYanWan**

### CODE REVIEW

File: dl/src/main/scala/com/intel/analytics/bigdl/utils/Table.scala [hunk 1]
```diff
@@ -42,6 +42,11 @@ class Table private[bigdl](
     }
   }
 
+  def clear(): Unit = {
```
Comment (qiuxin2012) [inline dl/src/main/scala/com/intel/analytics/bigdl/utils/Table.scala]: return itself

File: dl/src/main/scala/com/intel/analytics/bigdl/utils/Table.scala [hunk 2]
```diff
@@ -42,6 +42,12 @@ class Table private[bigdl](
     }
   }
 
+  def clear(): this.type = {
```
Comment (yiheng) [inline dl/src/main/scala/com/intel/analytics/bigdl/utils/Table.scala]: please add java doc for the new added public method

**Comment by yiheng** [comment]
Please merge commits into one commit

**Comment by JerryYanWan** [comment]
added

**Pull request merged**